src/ctelnet.h

```diff
@@ -282,6 +282,8 @@ public slots:
     QTimer* mpPostingTimer;
     bool mUSE_IRE_DRIVER_BUGFIX;
 
+    QNetworkReply* packageDownloadReply;
```

Comment:
I am not sure that you need to keep a pointer to the `QNetworkReply` as that can always (I *think*) be recovered from the `QNetworkRequest` - like what it did before...

... unless you need it for clean up purposes when you need to put:
```cpp
packageDownloadReply->abort();
mpProgressDialog->cancel();
packageDownloadReply->deleteLater();
```
into the cancel slot: `interfaceDownloadCancelled()` - and the `QProgressDialog::cancel()` will automatically hide the dialogue so that does not need to be done in that slot method...